wordcount.c

```diff
@@ -0,0 +1,34 @@
+/* Count the words in a string, now with user-input! The user types in a string, which then gets outputted to output.c, where it can be evaluated and the amount of words can be counted. - Amanda Cuevas */
```

Comment:
You didn't do the assignment quite how we wanted - you should have read `text.txt` in and iterated through the file, counting the words as you go along. That being said, nice job implementing this!